ui/app/styles/components/ui-wizard.scss

```diff
@@ -24,6 +28,11 @@
   bottom: $size-6;
   top: calc(4.5 * #{$size-6});
   overflow: auto;
+
+  @include until($desktop) {
```

Comment:
Should we do tablet? It seemed OK-ish on tablet size, but maybe half / half isn't very good.

Reply:
Yep, that would be better. Fixed